### PR TITLE
[IMP] panels: revamp the style of the side panels

### PR DESCRIPTION
**Pull request by hokolomopo**
## Description

Go through all the side panels and update their style to be prettier and more modern (and more odoo like).

Task: : [4080127](https://www.odoo.com/web#id=4080127&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/components/tables/table_styles_popover/table_styles_popover.ts [hunk 1]
```diff
@@ -19,11 +20,31 @@ export interface TableStylesPopoverProps {
...
+
+        &.selected {
+          border-bottom-color: #fff;
+          border-top-color: ${PRIMARY_BUTTON_BG};
+          color: ${TEXT_HEADING};
+        }
+      }
+    }
+
     .form-check-input {
```
Comment (LucasLefevre) [inline src/components/tables/table_styles_popover/table_styles_popover.ts:48]: this is unused

**Review by LucasLefevre — COMMENTED** [review]
Quite nice :)

I left a few screenshots in the task description, with broken stuff and stuff that still needs to be converted to the new style (can be later to merge this PR quickly).

File: src/components/side_panel/pivot/pivot_layout_configurator/add_dimension_button/add_dimension_button.xml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <templates>
   <t t-name="o-spreadsheet-AddDimensionButton">
-    <span class="btn btn-sm btn-link add-dimension" t-on-click="togglePopover" t-ref="button">
-      Add
-    </span>
+    <!-- ADM TODO not a fan of this one -->
```
Comment (LucasLefevre) [inline src/components/side_panel/pivot/pivot_layout_configurator/add_dimension_button/add_dimension_button.xml:3]: :eyes:

File: src/components/side_panel/split_to_columns_panel/split_to_columns_panel.ts [hunk 3]
```diff
@@ -104,6 +104,7 @@ export class SplitIntoColumnsPanel extends Component<Props, SpreadsheetChildEnv>
           errors.add(SplitToColumnsTerms.Errors[reason] || SplitToColumnsTerms.Errors.Unexpected);
       }
     }
+
```
Comment (LucasLefevre) [inline src/components/side_panel/split_to_columns_panel/split_to_columns_panel.ts:107]: useless diff in an untouched file

**Review by LucasLefevre — COMMENTED** [review]
:+1:

File: src/components/side_panel/pivot/pivot_layout_configurator/add_dimension_button/add_dimension_button.xml (hunk 2 above)
Comment (hokolomopo) [reply src/components/side_panel/pivot/pivot_layout_configurator/add_dimension_button/add_dimension_button.xml:3]: Francois was fine with the button so 🤷 I definitely need to remove the comment tho 😅

**Review by LucasLefevre — APPROVED** [review]
robodoo r+
:ok_hand:

**Pull request closed without merging**